champions/leading-cohort-calls.qmd

```diff
@@ -81,33 +81,80 @@ We develop these in a single "DigestsDrafts" Google Doc - with the AFSC Mentors
+-   Themed coworking was an experiment in Fall 2022 as a way to bring participants together across Cohorts. These were 1.5-hr sessions with Eli Holmes giving a lesson, followed by coworking in breakout rooms. Typically people stayed in the main room for Q & A.\
+-   Small group or 1:1 coworking is described in ([Butland & Penfold 2020](https://www.cscce.org/2020/02/04/online-co-working-partnerships-are-community-of-practice-in-action/)).
+
 ### Seaside Chats
 
 Champions teams will hold Seaside Chats, and we share suggested topics each Cohort Call.
 
+### Git / GitHub drop-in help
+
+After the GitHub Clinic lesson it might be helpful to offer optional drop-in help sessions for folks to get set up with Git and GitHub on their local machines.
+
+Eli Holmes recorded 4 short videos that people should watch before attending:
+
+1.  [Setting GitHub Desktop Preferences](https://youtu.be/0PVOPL6i3MQ) (2.5 mins)
+
+2.  [Cloning a repository](https://youtu.be/acIwQO1eOtw) (2 mins)
+
+3.  [Pulling and pushing changes to GitHub](https://youtu.be/cQgn3_w27_8) (3.5 mins)
+
+4.  [Turning an existing directory into a Git repository and pushing to GitHub](https://youtu.be/9BTpiTQPyu8) (2 min)
+
+In the Fall 2022 NMFS Cohorts we learned that many people needed help finding where Git was installed on their Windows machines, for example. Collaborative notes with screenshots were added in Google doc: OpenscapesCoworkingCross-Cohort \[ 2022-nmfs-fall \]
+
+::: {.callout-tip collapse="true"}
+## Example invitation to Git / GitHub drop-in sessions
+
+We've scheduled 30-minute drop-in sessions for those who want to get their computer connected to Git and GitHub. Drop in and we'll get you set up!
+
+What should I install beforehand? [GitHub Desktop](https://desktop.github.com/). This is a great option and is especially good if your GitHub repos have files from multiple programs (R, Python, Excel, Word, images, data files). If you install GitHub Desktop, you will have Git installed with that. If you don't get GitHub Desktop, then you will need to have Git installed.
+
+Once you have GitHub Desktop installed, watch the short videos from Eli Holmes to see how to get things set up.
+
+[Video 1](https://youtu.be/0PVOPL6i3MQ). Setting GitHub Desktop Preferences (2.5 mins)
+
+[Video 2](https://youtu.be/acIwQO1eOtw). Cloning a repository (2 mins)
+
+[Video 3](https://youtu.be/cQgn3_w27_8). Pulling and pushing changes to GitHub (3.5 mins)
+
+[Video 4](https://youtu.be/9BTpiTQPyu8). Turning an existing directory into a Git repository and pushing to GitHub (2 min)
+
+Give it a try and if you have any trouble getting connected, come to the drop-in sessions and we'll help!
+
+-   Date / time (click to add to your Google Cal)
```

Comment:
Ooh could you add instructions for "click to add to your google cal" somewhere?

Reply:
Yes! It's in this PR in a `Tip` under https://openscapes.github.io/approach-guide/champions/pre-cohort-tech.html#google-workspace

I'll merge this PR, then add a link from here

Reply:
done